compiler/luci-interpreter/src/kernels/Reverse.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ * Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
This file does not contain TensorFlow code.

Reply:
Okay I will remove it.